io.atom.electron.BaseApp.json

```diff
@@ -0,0 +1,229 @@
+                }
+            ]
+        },
+        {
+            "name": "gtk2",
+            "config-opts": ["--disable-gtk-doc", "--disable-introspection", "--disable-man",
+                            "--with-xinput=xfree" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/gtk+/2.24/gtk+-2.24.30.tar.xz",
+                    "sha256": "0d15cec3b6d55c60eac205b1f3ba81a1ed4eadd9d0f8e7c508bc7065d0c4ca50"
+                }
+            ]
+        },
+        {
+            "name": "gnome-themes-standard",
+            "config-opts": ["--disable-gtk3-engine"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/gnome-themes-standard/3.20/gnome-themes-standard-3.20.tar.xz",
+                    "sha256": "1cde84b34da310e6f2d403bfdbe9abb0798e5f07a1d1b4fde82af8e97edd3bdc"
+                }
+            ]
+        },
+        {
+            "name": "libnotify",
+            "config-opts": [ "--disable-static", "--disable-tests", "--disable-introspection" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/libnotify/0.7/libnotify-0.7.7.tar.xz",
+                    "sha256": "9cb4ce315b2655860c524d46b56010874214ec27e854086c1a1d0260137efc04"
+                }
+            ]
+        },
+        {
+            "name": "gvfs",
+            "cleanup": [ "/libexec/*", "/share/dbus-1/services/*", "/share/gvfs/mounts" ],
+            "config-opts": [ "--disable-hal", "--disable-gdu", "--disable-gcr", "--disable-obexftp",
+                             "--disable-avahi", "--disable-documentation", "--disable-admin" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/gvfs/1.28/gvfs-1.28.0.tar.xz",
+                    "sha256": "cf72fc0adf0ca702ead5b3fab3c1fa46b09678eb7c1290de7e30bb7cbaf5f704"
+                }
+            ]
+        },
+        {
+            "name": "libgpg-error",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.24.tar.bz2",
+                    "sha256": "9268e1cc487de5e6e4460fca612a06e4f383072ac43ae90603e5e46783d3e540"
+                }
+            ]
+        },
+        {
+            "name": "libgcrypt",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.7.3.tar.bz2",
+                    "sha256": "ddac6111077d0a1612247587be238c5294dd0ee4d76dc7ba783cc55fb0337071"
+                }
+            ]
+        },
```

Comment:
Same with libgcrypt. However, the yocto base is a bit weird in that it breaks the libgcrypt-config script:
```
$ /usr/bin/libgcrypt-config 
ERROR: /usr/bin/libgcrypt-config should not be used, use an alternative such as pkg-config
--should-not-have-used-/usr/bin/libgcrypt-config
```
So, you might have to patch the build to use pkg-config to find it.